crates/api/src/comment.rs

```diff
@@ -96,23 +86,20 @@ impl Perform for SaveComment {
     };
 
     if data.save {
-      let save_comment = move |conn: &'_ _| CommentSaved::save(conn, &comment_saved_form);
-      if blocking(context.pool(), save_comment).await?.is_err() {
+      let save_comment = CommentSaved::save(&&context.pool.get().await?, &comment_saved_form);
+      if save_comment.is_err() {
         return Err(ApiError::err("couldnt_save_comment").into());
       }
```

Comment:
A lot of these `is_err()` can probably be refactored a lot easier now. Its all preference really tho.